File: Regression/EP_Web/Careers/careers.py
from selenium import webdriver
from selenium.webdriver.common.keys import Keys
from selenium.webdriver.support.select import Select
import time
import xlrd
import os
import sys

if os.environ.get('USE_PHANTOM'):
    print("using PhantomJS")

    driver = webdriver.PhantomJS()
    driver.implicitly_wait(5)
    driver.set_window_size(1120, 550)
else:
    print("using Firefox")
    driver = webdriver.Firefox()

#EPWEB-2073

driver.get("https://pt.energypluscompany.com/company/careers.php")
elem = driver.find_element_by_id("logo")
print (driver.current_url)

if  driver.current_url != "https://careers.nrgenergy.com/":
    print("Failed - URL is %s" % driver.current_url)
    sys.exit(1)
else: 
    print("Passed")

### Close Session
driver.close()

File: Regression/EP_Web/Careers/careers.rb
require 'rubygems'
require 'watir-webdriver'
require 'win32ole' 
require 'csv'

#EPWEB-2073

browser = Watir::Browser.new :firefox
browser.goto 'https://pt.energypluscompany.com/company/careers.php'

if
browser.url == 'https://careers.nrgenergy.com/'
print "Careers Page - Passed", "\n"
else
print "Careers Page - Failed", "\n"
end

browser.close